src/test/java/org/folio/rest/api/RequestExpirationApiTest.java

```diff
@@ -736,7 +736,8 @@ public void canExpireOpenAwaitingWithNoHoldShelfExpirationDate()
 
   private void expireRequests() throws InterruptedException, ExecutionException, TimeoutException {
     CompletableFuture<Void> expirationCompleted = new CompletableFuture<>();
-    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx()).setHandler(res -> expirationCompleted.complete(null));
+    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx())
+      .onComplete(res -> expirationCompleted.complete(null));
     expirationCompleted.get(5, TimeUnit.SECONDS);
```

Comment:
Personally, I might use whitespace to separate this chain and the `get` below it